src/test/resources/mappings/request.json

```diff
@@ -0,0 +1,30 @@
+{
+  "mappings": [
+    {
+      "request": {
+        "method": "GET",
+        "url": "/circulation/requests/571b0a2c-8883-40b5-a449-d41fe6017083"
+      },
+      "response": {
+        "status": 200,
+        "body": "{\n    \"id\" : \"571b0a2c-8883-40b5-a449-d41fe6017083\"}",
+        "headers": {
+          "Content-Type": "application/json"
+        }
+      }
+    },
+    {
+      "request": {
+        "method": "POST",
+        "url": "/circulation/requests"
+      },
+      "response": {
+        "status": 200,
+        "body": "{\"id\": \"571b0a2c-8883-40b5-a449-d41fe6017083\"}",
```

Comment:
I would avoid using a global stub for POST. Especially if it is returning such a minimal response